terraform/modules/hub/hub_saml_engine.tf

```diff
@@ -57,6 +57,7 @@ data "template_file" "saml_engine_task_def" {
     certificates_config_cache_expiry = var.certificates_config_cache_expiry
     memory_hard_limit                = var.saml_engine_memory_hard_limit
     jvm_options                      = var.jvm_options
+    service_log_level                = var.service_log_level
```

Comment:
Can we update the task definition JSON file to pick up this variable `service_log_level `?